index.js

```diff
@@ -135,68 +127,153 @@ const runAction = (name, action, req, res, next) => {
+//. set to 200 (Ok).
+export const Json = value => (
+  Response.Respond (
+    200,
+    {'content-type': 'application/json'},
+    Body.Send (JSON.stringify (value)),
+  )
 );
 
-//# Render :: Number -> String -> Object -> Response a
+//# Render :: String -⁠> Object -⁠> Response a b
 //.
 //. Indicates a response to be rendered using a template. The first argument
-//. will be the response status code, the second is the path to the template
-//. file, and the third is the data to inject into the template. This uses
-//. Express' render method under the hood, so you can configure it globally
-//. with `app.set ('view engine', engine)` and `app.set ('views', path)`.
-export const Render = code => view => data => (
-  Response.Render (code, view, data)
+//. is the path to the template file, and the second is the data to inject into
+//. the template. This uses Express' render method under the hood, so you can
+//. configure it globally with `app.set ('view engine', engine)` and
+//. `app.set ('views', path)`.
+//.
+//. Unless overridden by [`withStatus`](#withStatus), the status code will be
+//. set to 200 (Ok).
+export const Render = view => data => (
+  Response.Respond (200, {}, Body.Render (view, data))
 );
 
-//# Redirect :: Number -> String -> Response a
+//# Redirect :: String -> Response a b
 //.
 //. Indicates a redirection. The first argument will be the response status
 //. code, and the second will be the value of the Location header.
-export const Redirect = code => location => (
-  Response.Redirect (code, location)
+//.
+//. Unless overridden by [`withStatus`](#withStatus), the status code will be
+//. set to 301 (Moved Permanently).
+export const Redirect = location => (
+  Response.Respond (301, {location}, Body.None)
 );
 
-//# Empty :: Response a
+//# Empty :: Response a b
 //.
 //. Indicates an empty response. The response status will be set to 204, and
 //. no response body or Content-Type header will be sent.
-export const Empty = Response.Empty;
+export const Empty = (
+  Response.Respond (204, {}, Body.None)
+);
 
-//# Next :: a -> Response a
+//# Next :: b -> Response a b
 //.
 //. Indicates that this middleware does not form a response. The supplied value
 //. will be assigned to `res.locals` and the next middleware will be called.
 export const Next = Response.Next;
 
+//# withHeader :: String -> String -> Response a b -> Response a b
+//.
+//. Sets the value of a header on the Response.
+export const withHeader = key => value => cata ({
+  Respond: (headers, code, body) => Response.Respond (
+    code,
+    {...headers, [key.toLowerCase ()]: value},
+    body,
+  ),
+  Next: Response.Next,
+});
+
+//# withoutHeader :: String -> Response a b -> Response a b
+//.
+//. Removes a header from the Response.
+export const withoutHeader = key => cata ({
+  Respond: (headers, code, body) => Response.Respond (
+    code,
+    {...headers, [key.toLowerCase ()]: undefined},
+    body,
+  ),
+  Next: Response.Next,
+});
```

Comment:
Before this PR, it was not possible to modify response headers through fluture-express